sympy/polys/rings.py

```diff
@@ -2531,6 +2531,124 @@ def compose(f, x, a=None):
 
         return poly
 
+
+    def coeff_wrt(self, sym, dg):
+        """
+        Coefficient of ``f`` with respect to ``x**deg``.
+
+        Treating ``f`` as a univariate polynomial in ``x`` this finds the
+        coefficient of ``x**deg`` as a polynomial in the other generators.
+
+        Parameters
+        ==========
+
+        p : sympy.Poly
+            The polynomial to compute the sparse coefficient for.
+        sym : int
+            The symbol to compute the coefficient for.
+        dg : int
+            The degree of the monomial to compute the coefficient for.
+
+        Returns
+        =======
+
+        sympy.Number
+            The sparse coefficient of the polynomial at the given symbol and degree.
```

Comment:
I think that "sparse coefficient" is a confusing term. Some other wording should be used to describe what is returned.